apps/vpnmanager/src/lib/email/sender.ts

```diff
@@ -25,9 +30,13 @@ export async function sendVpnKeyEmail({ recipient: to, key }: MailSender) {
       to,
       from,
       subject,
-      html: emailKeyTemplate(key),
+      html: emailKeyTemplate(key, name),
     };
     await sgMail.send(message);
+    Sentry.captureMessage(
+      `Email sent to "${to}" ${emailKeyTemplate("*hidden*", name)}`,
```

Comment:
```suggestion
      `Outline key sent`
```

We can then use [local scope](https://docs.sentry.io/platforms/javascript/enriching-events/scopes/#local-scopes) to set [user](https://docs.sentry.io/platforms/javascript/enriching-events/identify-user/) using `${to}` and [attachment](https://docs.sentry.io/platforms/javascript/enriching-events/attachments/#uploading-attachments) using `${emailKeyTemplate("*hidden*", name)}`.